### PR TITLE
Initial revision of Loom experiments

### PR DESCRIPTION
**Pull request by ljnelson**
This is just the config stuff to trigger it; more commits to follow.  The intent is that you can check this PR out and it will still build and run if you have JDK 11 but will be capable of being run under a Loom early build.

Signed-off-by: Laird Nelson <laird.nelson@oracle.com>

### CODE REVIEW

**Comment by ljnelson** [comment]
This (currently draft) PR is related to #1412.

File: microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java [hunk 1]
```diff
@@ -108,10 +109,34 @@ private void startServer(@Observes @Priority(PLATFORM_AFTER + 100) @Initialized(
 
         // make sure all configuration is in place
         if (null == jaxRsExecutorService) {
-            jaxRsExecutorService = ServerThreadPoolSupplier.builder()
+            Config serverConfig = config.get("server");
+            final java.lang.reflect.Method m;
+            if (serverConfig.get("virtual-threads").asBoolean().isPresent()) {
```
Comment (tomas-langer) [inline microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java:114]: `asBoolean().orElse(false)` - what if `virtual-threads: false` - you would execute as well

Comment (ljnelson) [reply microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java:114]: Yep; just a dumb typo in this draft PR

**Comment by ljnelson** [comment]
I removed my fork at one point so I'm going to close this and re-roll it with substantially the same changes.

**Comment by ljnelson** [comment]
Closing; watch for the new one!

**Comment by ljnelson** [comment]
See https://github.com/oracle/helidon/pull/2417

**Pull request closed without merging**